app/src/main/java/org/kiwix/kiwixmobile/KiwixWebViewClient.java

```diff
@@ -89,7 +90,15 @@ public void onPageFinished(WebView view, String url) {
             );
         view.addView(help);
         TextView contact = help.findViewById(R.id.welcome21);
-        contact.setText(StyleUtils.highlightUrl(contact.getText().toString(), CONTACT_EMAIL_ADDRESS));
+        FloatingActionButton share = help.findViewById(R.id.share);
+        share.setOnClickListener(shareview-> {
+          Intent sharingIntent = new Intent(Intent.ACTION_SEND);
+          sharingIntent.setType("text/plain");
+          sharingIntent.putExtra(Intent.EXTRA_SUBJECT,shareview.getContext().getString(R.string.sharing_message) );
```

Comment:
Spacing. Please no unnecessary white space. there are multiple issues in this file alone I am not going to point each one out.  commas are as they would be in english with "a, b"